packages/visual-editor/src/components/Dropzone/styles.module.css

```diff
@@ -9,20 +9,8 @@
   transition: outline 0.2s;
 }
 
-.isExpanded {
-  min-height: 80px;
-}
```

Comment:
Testing locally, it looks like it can be a little difficult to activate the dropzones inside of containers that are not very tall due to size of elements within.

For example adding a single line text component in a container, the hitboxes above and below the container on the root want to activate, making it harder to drop another component in the same container as the text component:

https://github.com/contentful/experience-builder/assets/8539634/2185a8bf-4d9a-4e14-b7a1-543b88ff45bd

Reply:
Nice catch! I've noticed some similar scenarios that I'm going to try and address today.

I'm going to try to add conditional logic to hitboxes to address this instead of expanding elements during a drag which lead to some unintended side effects with reparenting

Reply:
@primeinteger this will be done with a follow up ticket: https://contentful.atlassian.net/browse/ALT-504